packages/gatsby-source-drupal/src/gatsby-node.js

```diff
@@ -187,48 +206,62 @@ exports.sourceNodes = async (
     })
   })
 
-  // Download all files.
-  await Promise.all(
-    nodes.map(async node => {
-      let fileNode
-      if (
-        node.internal.type === `files` ||
-        node.internal.type === `file__file`
-      ) {
-        try {
-          let fileUrl = node.url
-          if (typeof node.uri === `object`) {
-            // Support JSON API 2.x file URI format https://www.drupal.org/node/2982209
-            fileUrl = node.uri.url
+  console.info(`Downloading remote files`)
+  downloadingFilesActivity.start()
+
+  // Download all files (await for each pool to complete to fix concurrency issues)
+  await asyncPool(concurrentFileRequests, nodes, async node => {
+    // If we have basicAuth credentials, add them to the request.
+    const auth =
+      typeof basicAuth === `object`
+        ? {
+            htaccess_user: basicAuth.username,
+            htaccess_pass: basicAuth.password,
           }
-          // Resolve w/ baseUrl if node.uri isn't absolute.
-          const url = new URL(fileUrl, baseUrl)
-          // If we have basicAuth credentials, add them to the request.
-          const auth =
-            typeof basicAuth === `object`
-              ? {
-                  htaccess_user: basicAuth.username,
-                  htaccess_pass: basicAuth.password,
-                }
-              : {}
-          fileNode = await createRemoteFileNode({
-            url: url.href,
-            store,
-            cache,
-            createNode,
-            createNodeId,
-            parentNodeId: node.id,
-            auth,
-          })
-        } catch (e) {
-          // Ignore
-        }
-        if (fileNode) {
-          node.localFile___NODE = fileNode.id
-        }
+        : {}
+    let fileNode = null
+    let fileUrl = ``
+    let url = {}
+
+    if (node.internal.type === `files` || node.internal.type === `file__file`) {
+      fileUrl = node.url
+
+      // If node.uri is an object
+      if (typeof node.uri === `object`) {
+        // Support JSON API 2.x file URI format https://www.drupal.org/node/2982209
+        fileUrl = node.uri.url
+      }
+
+      // Resolve w/ baseUrl if node.uri isn't absolute.
+      url = new URL(fileUrl, baseUrl)
+
+      // Create the remote file from the given node
+      try {
+        fileNode = await createRemoteFileNode({
+          url: url.href,
+          store,
+          cache,
+          createNode,
+          createNodeId,
+          parentNodeId: node.id,
+          auth,
+        })
+      } catch (err) {
+        // Ignore
+        console.error(err)
```

Comment:
do we want to show an error? We ignored this before. If we want this I recommend using report.error and remove the comment

Reply:
It would be nice (if anything was to fail) to have the visibility of what went wrong.

Will change this to `reporter.error`